config/yurt-app-manager/samples/uniteddeployment/uniteddeployment_deployment_test.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps.openyurt.io/v1alpha1
+kind: UnitedDeployment
```

Comment:
change file name from `uniteddeployment_deployment_test.yaml` to `uniteddeployment_deployment.yaml`.